bentoml/repository/__init__.py

```diff
@@ -118,6 +122,21 @@ def __init__(self, base_url):
         self.base_url = base_url
         self.uri_type = BentoUri.S3
 
+        parse_result = urlparse(base_url)
+        bucket = parse_result.netloc
+        base_path = parse_result.path
+
+        s3_client = boto3.client("s3")
+
+        try:
+            filename = uuid.uuid4().hex
+            s3_path = os.path.join(base_path, filename)
+            s3_client.upload_file(Filename=filename, Bucket=bucket, Key=s3_path)
```

Comment:
Is this for testing users' write-permission? I think it's ok to just be optimistic here and only raise when it actually run into problems uploading things to s3

Reply:
yes, ok, will remove this.